eos-prune-printers

```diff
@@ -36,9 +36,22 @@ done
 systemctl stop cups
 
 # remove stamp file from eos-config-printer (from 4.0 use driverless printers)
-rm -rf /var/lib/eos-config-printer
+LIB_CONFIG_PRINTER=/var/lib/eos-config-printer
+if [ -d $LIB_CONFIG_PRINTER ]; then
+	echo Removing $LIB_CONFIG_PRINTER
+	rm -rf $LIB_CONFIG_PRINTER
+else
+	echo $LIB_CONFIG_PRINTER does not exist
+fi
+
 # clean up the residual print cache files
-rm -rf /var/cache/cups/*
+CAHCE_CUPS=/var/cache/cups
+if [ -d $CAHCE_CUPS ]; then
+	echo Removing $CAHCE_CUPS/*
```

Comment:
What do you hope that this will print? Do you hope for:

```
Removing /var/cache/cups/*
```

or do you hope for

```
Removing /var/cache/cups/foo /var/cache/cups/bar [… and so on for each file in that directory]
```
?

Reply:
(As written, it will output the former if `/var/cache/cups` is empty, and the latter if it contains one or more files. This is related to the shellcheck warning below.)

Reply:
I'd like simple `/var/cache/cups/*`